sqlscripts/0003-jaeger-operations.sql

```diff
@@ -8,5 +8,5 @@ AS SELECT
     service,
     operation,
     count() as count
-FROM jaeger_index_local
+FROM %s
```

Comment:
Is it possible to add a comment to this file. I would like to mention here that here goes `jaeger_index_local`